### PR TITLE
Multiple success callbacks

### PR DESCRIPTION
**Pull request by randdusing**
Mostly made improvements to multiple success callbacks and created a demo.

### CODE REVIEW

**Comment by gortok** [comment]
Can you squash these commits using `rebase -i`?  It's hard to follow what's going on with so many commits.  From what I see so far, I like what you did.

**Comment by randdusing** [comment]
I did the rebase, but tt still looks like a mess since I removed node_modules. I figure you can just npm install. Should I go ahead and remove the references to my branch? The homepage and bower install needs updating.

**Comment by randdusing** [comment]
Do you think we'll be able to merge this? I have some plugin updates coming soon, and would like to keep the Angular wrapper up to date as well.

**Comment by gortok** [comment]
There are merge conflicts. If you can clean up the commits and rebase off the current state of master I'd love to include these fixes.

**Comment by randdusing** [comment]
Ok, I'm going to close this one and will submit a new pull request when the plugin and wrapper are updated.
